quickstarts/redmine/config.yml

```diff
@@ -0,0 +1,45 @@
+
+
+slug: redmine
+description: |
+  ## Comprehensive monitoring quickstart for Redmine
+  We can monitor your Redmine application in a complex environment and keep you on top of performance with our Ruby agent.
+  ## Why monitor Redmine?
+  The performance monitoring tool for Redmine helps you monitor the entire infrastructure and identifies the critical issues in your Redmine application. Redmine performance monitoring enables you to get a complete overview of Redmine errors and know when and where your Redmine application slows down. Some key components are:
```

Comment:
Delete everything and replace with:

"When you monitor your Redmine app's performance, you'll get a complete overview of your errors and when and where your Redmine application is slowing down. Specifically, you can use this to:"